docs/contributing/README.md

```diff
@@ -1,9 +1,142 @@
-# Welcome to the Knative community
+# About Knative
 
-Knative is an open source project that anyone in the community can use, improve, and enjoy.
-We'd love you to join us!
+Knative is an open source project with an active development community.
+The project was started by Google but has contributions from a growing number of industry-leading companies. With a clear project scope, lightweight governance model, and clean lines of
+separation between pluggable components, the Knative project establishes an
+efficient contributor workflow.
 
-In this section:
+Knative is a diverse, open, and inclusive community.
 
-- [Contribute to Knative](contributing.md): helpful tips for how to get started contributing to Knative.
-- [About the Knative community](about.md): information about the Knative community and how it's run.
+Knative is designed for different personas:
+
+![Diagram that displays different Audiences for Knative](images/knative-audience.svg)
+
+Knative components offer developers Kubernetes-native APIs for deploying
+serverless-style functions, applications, and containers to an auto-scaling
+runtime.
+
+To join the conversation, head over to the
+[Knative users](https://groups.google.com/d/forum/knative-users) Google group.
+
+Knative components are intended to be integrated into more polished products
+that cloud service providers or in-house teams in large enterprises can then
+operate.
+
+Any enterprise or cloud provider can adopt Knative components into their own
+systems and pass the benefits along to their customers.
```

Comment:
```suggestion
systems and pass the benefits along to their developers, platform teams, and customers.
```

Reply:
Added a link to ADOPTERS.MD here to highlight people who have adopted the project.